UI/auth-oauth.cpp

```diff
@@ -91,7 +93,7 @@ void OAuthLogin::urlChanged(const QString &url)
 	if (code_idx == -1)
 		return;
 
-	if (url.left(22) != "https://obsproject.com")
+	if (url.startsWith(OAUTH_BASE_URL))
```

Comment:
It should be `(!url.startsWith(OAUTH_BASE_URL))` or I'm missing something.

Reply:
No you're right, I forgot to change that.